app/src/main/java/com/android/streetworkapp/ui/progress/Progression.kt

```diff
@@ -370,3 +386,18 @@ sealed class DashboardStateProgression {
 
   data object Achievement : DashboardStateProgression()
 }
+
+fun exerciseNameToIcon(name: String): Int {
+  return when (name) {
+    "Push-ups" -> R.drawable.train_pushup
+    "Dips" -> R.drawable.train_dips
+    "Burpee" -> R.drawable.train_burpee
+    "Lunge" -> R.drawable.train_lunge
+    "Planks" -> R.drawable.train_planks
+    "Handstand" -> R.drawable.train_hand_stand
+    "Front lever" -> R.drawable.train_front_lever
+    "Flag" -> R.drawable.train_flag
+    "Muscle-up" -> R.drawable.train_muscle_up
+    else -> R.drawable.handstand_org
```

Comment:
Here too, instead of using hard coded strings just use the resource string to do the mapping.

Reply:
Done, thank you!